vividus/src/main/java/org/vividus/bdd/steps/FileSteps.java

```diff
@@ -53,11 +54,12 @@
      */
     @When("I create temporary file with name `$name` and content `$content` and put path to $scopes variable"
             + " `$variableName`")
-    public void saveResponseBodyToFile(String name, String content, Set<VariableScope> scopes, String variableName)
+    public void createTemporaryFile(String name, DataWrapper content, Set<VariableScope> scopes, String variableName)
```

Comment:
could you please add an integration test for the case with binary data

Reply:
it's added as a part of Avro test